master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -43,6 +43,7 @@ spec:
 | logSeveritySys | The log severity above which logs are sent to the syslog. Set to `NONE` for no logging to syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) | string | `INFO` |
 | logSeverityFile| The log severity above which logs are sent to the log file. | Same as `logSeveritySys` | string | `INFO` |
 | logSeverityScreen | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | string | `INFO` |
+| ignoreLooseRPF | Allows Felix to run on systems with loose reverse path forwarding (RPF). | boolean | boolean | `false` |
```

Comment:
How about a big fat warning explaining why that's a bad idea?

Reply:
Hmm this is a table format, not sure where the big fat warning would go. Any suggestions, @fasaxc @emanic ? (Also, I don't really know why this is a bad idea :D )

Reply:
We rely on the kernel's RPF check to prevent pods from spoofing each other.  If you allow "loose" mode, pods on the same host can happily use each other's IP addresses.

In the past, we've had very wide tables but I think we've been cleaning that up so I'll let @emanic make a suggestion.

Reply:
@gunjan5 @fasaxc I am not sure how the rows are ordered. I might think an alphabetical ordering could be one approach, to have some ordering that makes sense. However, it is possible that we have another reason for the current ordering of the columns.

One way to handle this might be an asterisk beside `ignoreLooseRPF` and then at the bottom of the table, you also have an asterisk (need to escape it with `\`) where you explain why this is a bad idea or at least, some of the reasons why you might _not_ want to do it.

You can see a page like this here https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy So you can see the syntax if you look at the source with the escapes, etc.

Reply:
Ok thanks @fasaxc , and good idea @emanic , I'll update the PR with those suggestions